cmd/helm/testdata/testcharts/chart-with-template-lib-dep/charts/common/templates/_ingress.yaml

```diff
@@ -4,7 +4,7 @@ kind: Ingress
 {{ template "common.metadata" . }}
   {{- if .Values.ingress.annotations }}
   annotations:
-    {{ include "common.annote" .Values.ingress.annotations | indent 4 }}
+    {{ include "common.annotate" .Values.ingress.annotations | indent 4 }}
```

Comment:
Since this is a test chart, its content shouldn't matter, that said, it isn't a problem for me to drop this (or anything else).